FordLabs/PeopleMover
# Fix flaky calendar tests

## Issue
Connects #294 

## What was done
- [x] Deleted the flaky calendar unit tests and replaced them with cypress calendar tests.
- [x] Consolidated the functionality that sets up and resets the flipping sweet space (created LocalDataGenerator in the backend)

## How to test
1. Run the cypress tests and ensure that the calendar tests pass
2. Run the calendar unit tests and ensure they are now consistently passing in github actions
